lib/jets/resource/lambda/function.rb

```diff
@@ -136,6 +136,8 @@ def finalize_properties!(props)
 
     def get_layers(runtime)
       return nil unless runtime =~ /^ruby/
+      return Jets.config.lambda.layers if Jets.config.gems.disable
+      
```

Comment:
@tongueroo maybe the best is to add another option for this?

```ruby
Jets.config.gems.layer = false
```

Reply:
Think it's fine to use the same option, one less option to remember.